docs/send-data/opentelemetry-collector/faq.md

```diff
@@ -218,10 +83,8 @@ If you see a log containing `"listen tcp :8888: bind: address already in use"` i
 2022-11-17T11:22:17.733+0100    error   service/collector.go:156        Asynchronous error received, terminating process        {"error": "listen tcp :8888: bind: address already in use"}
 ```
 
-this means that the `8888` port on the machine is busy.
-It is possible that there is another collector already running on the host,
-but it can also be any other process.
-To find out what process is using the port, run the following command:
+This means that the `8888` port on the machine is busy. It is possible that there is another collector already running on the host,
+but it can also be any other process. To find out what process is using the port, run the following command:
```

Comment:
```suggestion
This means that the `8888` port on the machine is busy. It is possible that there is another collector already running on the host, but it can also be any other process. To find out what process is using the port, run the following command:
```